code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -1,6 +1,6 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
-	damage = 50
+	damage = 30
```

Comment:
Personally, I think you should move the niche of these slugs into something else. Maybe give it a higher wounding chance and embed chance to give some use case for slugs over other ammo types? A flat damage reduction is only going to have people look elsewhere to bring people down. Namely, to beanbags, which also do what I suggest and also do more damage to boot.

Reply:
i'd rather we kill slugs entirely because it's fucking stupid that we take the expected design of shotguns (spread of pellets) and make it stupid and boring

Reply:
I agree but I also would like to point out we have one other slugthrower in sec. The .38 revolver. Shoots faster, does as much damage, embeds and wounds. Pretty nasty, jusy eclipsed by shotguns most of the time.

Also, bartenders use beanbags, so would you prefer them using rubbershot? Or not have a gun at all?

Reply:
using beanbags is fine by me

Reply:
Personally I think that shotguns mandatorily being a scattershot weapon is a dumb standard. One of the coolest things about shotguns is the variety of ammo types.

Also yeah with wounds, slugs are very dangerous because they cause serious bleeding; not just for their brute damage. I think that wound-focused projectile damage is a cooler system anyway, doing away with 2-shots-ur-ded-lmao would be an improvement, in favour of HEY FIRST AID IS USEFUL!